src/files/source/metadata.php

```diff
@@ -1429,6 +1468,18 @@
     'server.read' => [
         'description' => 'Allows reading of the server',
     ],
+    'server.read-all' => [
+        'description' => 'Allows reading of extended data of the server',
+    ],
```

Comment:
If I get a `server.read-all` permission, does it mean I also get `server.wizzard-read`, `server.read-legend`?

If yes – then `server.read-all` is a role, not a permission.

If no – then the name should be changed.

Reply:
No. Your proposition? This permission used in
hipanel-module-server:
    grid/ServerGridView.php
    grid/ServerRepresentations.php
    views/server/_search.php

Reply:
- `server.read-financial-info`
- `server.read-system-info`